provider-directory/src/main/resources/application.properties

```diff
@@ -1,4 +1,7 @@
 ppms.url=unset
+vler.key.public=unset
+vler.key.private=unset
+vler.url=unset
```

Comment:
For the make-configs script to work, I think the two trust-store variables need to be `unset` in here as well.

Reply:
Placed them under this comment spot